src/compiler/token.cr

```diff
@@ -35,19 +35,24 @@ module Lucid::Compiler
       PlusAssign        # +=
       MinusAssign       # -=
       StarAssign        # *=
-      SlashAssign       # /=
       DoubleStarAssign  # **=
+      SlashAssign       # /=
       DoubleSlashAssign # //=
 
-      IsA
-      RespondsTo
+      IsA        # is_a?
+      RespondsTo # responds_to?
```

Comment:
`as`, `as?`, and `nil?` are pseudo methods implemented by the compiler as well

Reply:
I left them out as they haven't been implemented yet — really I should have removed most of the keywords but they were added into the lexer while the parser was still in early stages. As a side note: `nil?` will not be included here, I will make a separate issue soon explaining why.